### DISCUSSION TITLE
SPECFEM Logo Design

### DISCUSSION THREAD

**Discussion by bch0w** [SPECFEM/specfem3d#1549] [General]
We're looking for a SPECFEM organization logo!

During the SPECFEM developer's workshop last week we decided that it would be great to have a SPECFEM logo that is independent of any individual codebase, and that we can use for the organization profile picture, and any future posters, presentations etc. 

It would be great to get submissions from the User community which we might be able to select from or vote on. If you're feeling artistic, please submit a design here! Or if you have a good idea but not enough time or talent, you can post it here and see if anyone will sketch it out for you. 

Things we might be looking for in a logo include: original, simple, scalable, and representative of the community as a whole. Consider that this logo might be placed anywhere from large posters to small zoom presentations.

**Comment by ljhwang** [comment]
Not really related to the logo design, but is part of the branding that the specfems are open source? if so, maybe we change our tag line on specfem.org to (adding the open source part):
           A family of open source spectral-element method software codes for computational seismology.

Are there other concepts to include in a logo as part of a brand identity? Just trying to generate some ideas that can distinguish the project from other SEM codes!

**Reply by danielpeter** [reply]
good suggestion, I added open-source to the tag line. it's an integral part of the project.

**Comment by ljhwang** [comment]
Logo thought. Instead of using cubes or meshes like some SEM projects for logos, how about basing it on a "banana" kernel not to be confused with the amazon smile. Then use simply an S or S123 to represent the concept and S1, S2, S3 etc to represent the individual packages?  Again not to be confused with amazon EC2, the elements could vary in size, color, and placement.

**Reply by bch0w** [reply]
I like this! i.e., focusing on the 'adjoint' capabilities of the codes. I think it's also useful to stress to forward capabilities, perhaps adding a waveform in there as well. I was sketching up an idea, will try to work this into the next iteration

**Comment by foivoskar** [comment]
Hello, during the last few days I'm working on an idea for a logo. I had the chance to use also some feedback in person in Toronto and at the moment I have the versions that I attach. If you like it I can work more on the details, and if you'd like to improve details any kind of feedback would be very valuable. It's still a draft that needs to be polished to look more professional, however, to do this, it's important to know which direction to improve.

![specfemlogo_color_draft_1](https://user-images.githubusercontent.com/53915305/200082927-3bc2d1cb-684d-4d64-b1f3-571f686dfc9e.png)

**Reply by casarotti** [reply]
I love it
my 2 cents is for a simpler version ... only letter and wave

**Reply by ljhwang** [reply]
Love how you incorporated the seismogram. Can you explain the graphic to me?

**Reply by foivoskar** [reply]
The drawing around C is meant to be representative of the chunks of the 3D_globe version, however somehow minimal without a very detailed shape and no closed frames for the "elements".

**Reply by foivoskar** [reply]
Concerning the "only letters" version, I think that just removing the graphic doesn't improve the whole thing, however, with some other modifications, it can probably work better.

![comparison_1](https://user-images.githubusercontent.com/53915305/200087515-72e7a422-efc2-4d0c-a563-e41f88019f3b.png)

**Reply by williameaton** [reply]
This looks great! One thought I had for incorporating the kernels would be to make the letter D (for 2D or 3D ... i guess this wouldnt work for globe) out of the banana kernel shape? Or the letter C in specfem .... It may look a bit tacky, but just an idea!

**Comment by jwbishop** [comment]
Hi All,

You could perhaps integrate the cubed sphere concept into the "C" in
SPECFEM if you mirror it horizontally.

Best,
Jordan

**Comment by foivoskar** [comment]
Hi all, I tried to incorporate the suggestions that all of you made in the previous draft, here is a new version to continue the discussion

![specfem_logo_draft2_presentation](https://user-images.githubusercontent.com/53915305/200126808-3ebd5931-700a-4c10-accc-eb15f224ee35.png)

**Comment by elifo** [comment]
Hello everyone! A nearly hexahedron house after reading the homepage text — I couldn't stop the child in me :)  

<img width="1027" alt="Screen Shot 2022-11-05 at 2 45 48 PM" src="https://user-images.githubusercontent.com/26336068/200142712-09fe3f8b-5868-497c-85d2-2b1cf7499166.png">

**Comment by jpampuero** [comment]
I second Jordan's suggestion for Foivos' logo: maybe the "C" could be carved out of the global Earth SPECFEM3D mesh by removing the core and three contiguous chunks, then rotating it.
Pablo

**Reply by foivoskar** [reply]
Oh! Now I see what you're talking about. Actually, I tried to represent this concept on the second version. Taking the exact mesh and put it on the logo is not very good idea, because it is not going to be scalable. I try thick lines because it should also work when converted to a thumbnail, avatar, etc.

**Comment by danielpeter** [comment]
one more idea, just to have some more inspiration:

![specfem-logo](https://user-images.githubusercontent.com/5815257/200572932-73b1fda0-8bec-49e2-acda-fc23ca8592b7.png)

**Comment by bch0w** [comment]
Throwing in another idea I've been kicking around in my head since the workshop. I'm a fan of simple icons that can be used standalone and incorporated into a bigger logo. The motivation here was to include all flavors of SPECFEM in the logo, so the three pieces are a global chunk, cartesian block and 2D plane. Relative line widths, font choice, etc. will need some tweaking. It also might be interesting to throw in a wiggle or kernel to fill up some space. I've included a few versions, with and without text.

![sem_logo_text_simple](https://user-images.githubusercontent.com/23055374/204417158-96facbe6-72f6-4939-b5a0-164abb874ae1.png)
![sem_logo_simple_comparison](https://user-images.githubusercontent.com/23055374/204417608-e1af6ebe-79b8-4def-9906-55c4a8305b3e.png)
![sem_logo_complex_text](https://user-images.githubusercontent.com/23055374/204417176-5fab7a7d-aa22-4b76-b1b8-1abb7144bd71.png)
![sem_logo_complex_comparison](https://user-images.githubusercontent.com/23055374/204418171-84414000-5687-4a3a-9fe1-f42da2616305.png)
